.github/workflows/auto-review.yml

```diff
@@ -0,0 +1,38 @@
+name: Auto Assign Reviewer
+
+on:
+  pull_request_target:
+    types: [opened, ready_for_review]
+
+permissions:
+  pull-requests: write
```

Comment:
I think this is OK as is, though Chris added a Github app for the releases so that's another option for these kinds of workflows too.